src/Microsoft.AspNet.Razor/TagHelpers/TagHelperUsageDescriptor.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.TagHelpers
+{
+    /// <summary>
+    /// A metadata class describing how to use a tag helper at design time.
```

Comment:
The part about `at design time` has nothing to do with this class; that choice is elsewhere. Suggest `A metadata class containing information about tag helper use.`